testsuite/MDAnalysisTests/coordinates/test_h5md.py

```diff
@@ -493,9 +358,325 @@ def test_no_units(h5md_file, ref, tmpdir):
                 del g['particles/trajectory/position/time'].attrs['unit']
                 del g['particles/trajectory/box/edges/value'].attrs['unit']
         with pytest.raises(ValueError):
-            u = mda.Universe(TPR_xvf, outfile, format="H5MD")
+            u = mda.Universe(TPR_xvf, outfile)
```

Comment:
`assert` something here

Reply:
should this be asserting something? I thought it was just trying to catch the value error

Reply:
You're right, I must have overlooked the raises.